### PR TITLE
Django-4.2 update

### PR DESCRIPTION
**Pull request by Evan-CTL**

### CODE REVIEW

File: brownfield_django/settings_shared.py [hunk 1]
```diff
@@ -35,8 +35,8 @@
     'registration',
     'storages',
     'brownfield_django.main',
-    'django_markwhat',
-    'contactus'
+    'contactus',
+    'markdownify',
```
Comment (ndittren) [inline brownfield_django/settings_shared.py:39]: I believe this is supposed to be `markdownify.apps.MarkdownifyConfig` according to their doc: https://django-markdownify.readthedocs.io/en/latest/install_and_usage.html

File: brownfield_django/mixins.py [hunk 2]
```diff
@@ -4,8 +4,7 @@
 
 from django.contrib.auth.decorators import login_required, user_passes_test
 from django.contrib.auth.models import User
-from django.http.response import HttpResponseNotAllowed, HttpResponse, \
-    HttpResponseForbidden
+from django.http.response import HttpResponse, HttpResponseForbidden
 from django.utils.decorators import method_decorator
 from django.views.decorators.csrf import csrf_exempt
```
Comment (ndittren) [inline brownfield_django/mixins.py:10]: Same thing here, we can test like this: https://github.com/ccnmtl/writlarge/blob/24b9bc28b9b2e7643221c4bd133806271724d4c1/writlarge/main/mixins.py#L17

File: brownfield_django/settings_shared.py (hunk 1 above)
Comment (Evan-CTL) [reply brownfield_django/settings_shared.py]: Done

File: brownfield_django/mixins.py (hunk 2 above)
Comment (Evan-CTL) [reply brownfield_django/mixins.py:11]: Done

**Review by ndittren — APPROVED** [review]

**Pull request merged**